### PR TITLE
SDK v1 first PR

### PR DESCRIPTION
**Pull request by ekenigs**

### CODE REVIEW

File: packages/builder/fixtures/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from './builderPramsMock';
```
Comment (mmaurello) [inline packages/builder/fixtures/index.ts:1]: typo in this filename

File: packages/config/src/configs/acala.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      },
+      extrinsic: ExtrinsicBuilder().xTokens().transfer(),
+    }),
+    new AssetConfig({
+      asset: ausd,
+      balance: BalanceBuilder().tokens().accounts(),
+      destination: moonbeam,
+      destinationFee: {
+        amount: FeeBuilder().assetManager().assetTypeUnitsPerSecond(),
+        asset: aca,
```
Comment (mmaurello) [inline packages/config/src/configs/acala.ts:30]: `ausd` transactions are paid by `ausd` itself in the destination, not `aca`, we don't have to send acala along to pay for the fees

File: packages/config/src/configs/karura.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      },
+      extrinsic: ExtrinsicBuilder().xTokens().transfer(),
+    }),
+    new AssetConfig({
+      asset: ausd,
+      balance: BalanceBuilder().tokens().accounts(),
+      destination: moonriver,
+      destinationFee: {
+        amount: FeeBuilder().assetManager().assetTypeUnitsPerSecond(),
+        asset: kar,
```
Comment (mmaurello) [inline packages/config/src/configs/karura.ts:30]: Same as with acala. `ausd` transactions are paid by `ausd` itself in the destination, not `kar`

File: packages/config/src/configs/kinitsugi.ts [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  BalanceBuilder,
+  ExtrinsicBuilder,
+  FeeBuilder,
+} from '@moonbeam-network/xcm-builder';
+import { kbtc, kint } from '../assets';
+import { kinitsugi, moonriver } from '../chains';
+import { AssetConfig } from '../types/AssetConfig';
+import { ChainConfig } from '../types/ChainConfig';
+
+export const kinitsugiConfig = new ChainConfig({
```
Comment (mmaurello) [inline packages/config/src/configs/kinitsugi.ts:11]: typo in the name of the const and in also in the name of the file, in general replace all the `kinitsugi` with `kintsugi` in the whole project

File: packages/sdk/src/getTransferData.ts [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+          ? destination.fee.toBig()
+          : Big(0),
+      );
+
+      return source.balance.copyWith({
+        amount: result.lt(0) ? 0n : BigInt(result.toString()),
+      });
+    },
+    /**
+     * Right not it will be always true
```
Comment (mmaurello) [inline packages/sdk/src/getTransferData.ts:64]:
```suggestion
     * Right now it will be always true
```

**Review by mmaurello — CHANGES_REQUESTED** [review]
The code looks good in general, nice work. I just found a few inconsequential typos and also a couple of issues in the config. 
With the new version changes in the dapp we'll test more thoroughly

File: packages/config/src/configs/kinitsugi.ts (hunk 4 above)
Comment (ekenigs) [reply packages/config/src/configs/kinitsugi.ts:11]: this is fixed in my next PR

**Review by mmaurello — APPROVED** [review]

**Pull request merged**